clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -1358,14 +1358,14 @@ void CodeGenModule::Release() {
         TheModule.getOrInsertNamedMetadata("opencl.spir.version");
     SPIRVerMD->addOperand(llvm::MDNode::get(Ctx, SPIRVerElts));
     // We are trying to look like OpenCL C++ for SPIR-V translator.
```

Comment:
The language mode is called "C++ for OpenCL".
```suggestion
    // We are trying to look like C++ for OpenCL for SPIR-V translator.
```